File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    const string = s.toLowerCase();
    
    let pCnt = 0; let yCnt = 0;
    
    [...string].forEach(s => {
        if (s === 'p') pCnt += 1;
        else if (s === 'y') yCnt += 1;
    })
    
    return yCnt === pCnt ? true : false;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;

        s = s.toLowerCase();
        int pCnt = 0;
        int yCnt = 0;
        
        for (int i = 0; i < s.length(); i++) {
            char l = s.charAt(i);
            if (l == 'p') pCnt += 1;
            else if (l == 'y') yCnt += 1;
        }
    

        return pCnt == yCnt ? true : false;
    }
}